ordered_associative_task/student/include/impl.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 #include <map>
 
+extern std::map<size_t,size_t> mapConverter;
```

Comment:
why do you need extern here?

Reply:
I used extern in order to declare the map in the header and define it in the source. Actually, I can declare and define it only in the cpp file, right?

Reply:
yes